src/main/java/com/woowacamp/soolsool/core/member/domain/vo/MemberMileage.java

```diff
@@ -35,4 +31,8 @@ private void validateIsNotNull(final BigInteger mileage) {
             throw new SoolSoolException(MemberErrorCode.NO_CONTENT_MILEAGE);
         }
     }
+
+    public MemberMileage charge(final BigInteger amount) {
+        return new MemberMileage(this.mileage.add(amount));
+    }
```

Comment:
```suggestion
    public MemberMileage charge(final MemberMileage amount) {
        return new MemberMileage(this.mileage.add(amount.mileage));
    }
```

MemberMileage 간에 메시지를 전달하는 것이 좋아보여요. 
예를 들어 BigInteger의 값을 증가시키기 위해서 BigInteger.add()를 하는데 매개 변수로 String을 넘기진 않으니까요.

Reply:
구우욷~~